curvefs/src/client/inode_wrapper.cpp

```diff
@@ -93,7 +148,28 @@ CURVEFS_ERROR InodeWrapper::Sync() {
     return CURVEFS_ERROR::OK;
 }
 
-CURVEFS_ERROR InodeWrapper::Refresh() {
+void InodeWrapper::FlushAttrAsync() {
+    if (dirty_) {
+        LockSyncingInode();
+        auto *done = new UpdateInodeAsyncDone(shared_from_this());
+        metaClient_->UpdateInodeAsync(inode_, done);
+        dirty_ = false;
+    }
+}
+
+void InodeWrapper::FlushS3ChunkInfoAsync() {
+    if (!s3ChunkInfoAdd_.empty()) {
+        LockSyncingS3ChunkInfo();
+         auto *done = new GetOrModifyS3ChunkInfoAsyncDone(shared_from_this());
+        metaClient_->GetOrModifyS3ChunkInfoAsync(
+            inode_.fsid(), inode_.inodeid(), s3ChunkInfoAdd_,
+            done);
+        s3ChunkInfoAdd_.clear();
```

Comment:
why not move it when calling `GetOrModifyS3ChunkInfoAsync`?

Reply:
s3ChunkInfoAdd_ should clear here， so that new s3chunkinfo can add to it when new write come